packages/drawing-response/src/drawing-response/drawable-palette.jsx

```diff
@@ -95,6 +99,9 @@ const styles = theme => ({
     marginRight: theme.spacing.unit * 2,
     marginTop: theme.spacing.unit * 2
   },
+  blackColorItem: {
+    color:'#FFFFFF',
```

Comment:
Check if there is any color constant for this, don't use hex code.
color.text() should be it.